data/core/units.cfg

```diff
@@ -299,7 +299,7 @@ The eastern provinces of Wesnoth, known as the Clan Homelands, have a geography
         male_name= _ "race^Orc"
         female_name= _ "race+female^Orc"
         plural_name= _ "race^Orcs"
-        description= _"In appearance, orcs are half men and half beasts. They are taller, sturdier and stronger than humans. They are warlike, savage, and cruel by nature. Their blood is darker and thicker than that of normal humans and they have little care for personal hygiene or their personal appearance. Although Orcs are violent even among themselves, they are pack-oriented; an orc never travels long or lives in groups smaller than half a dozen.
+        description= _"In appearance, orcs resemble humans but with some bestial features. They are taller, sturdier and stronger than humans. They are warlike, savage, and cruel by nature. Their blood is darker and thicker than that of humans, and they have little care for personal hygiene or their personal appearance. Although Orcs are violent even among themselves, they are pack-oriented; an orc never travels long or lives in groups smaller than half a dozen.
```

Comment:
"never travels long" sounds wrong to me, "never travels long distances" or  "never travels alone" both sound better

Reply:
Oh, good point.